language/units_en.lua

```diff
@@ -412,6 +412,11 @@ return {
 				corroy = "Enforcer",
 				corsb = "Dam Buster",
 				corscreamer = "Screamer",
+				corscavdrag="Scav Dragon's Teeth",
+				corscavfort="Scav fortification wall",
+				corscavdtm="Scav missile wall",
+				corscavdtl="Scav Dragon's claw",
+				corscavdtf="Scav Dragon's Maw",
```

Comment:
"Scav" should be removed, as these are not technically Scavenger units - those would be `corscavdrag_scav`, `corscavfort_scav`, etc. Scavenger unit names are then handled separately, see line 11.